proto/stride/auction/query.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package stride.auction;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "stride/auction/params.proto";
+import "stride/auction/auction_pool.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v5/x/auction/types";
+
+// Query defines the gRPC querier service.
+service Query {
+
+  // Parameters queries the parameters of the module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/stride/v5/auction/params";
+  }
+
+  // Queries a list of AuctionPool items.
+  rpc AuctionPool(QueryGetAuctionPoolRequest)
```

Comment:
```suggestion
  rpc AuctionPool(QueryAuctionPoolRequest)
```
Common practice for defining query req & resp msgs are Query[rpc name]Request/Response